polaris/ocean/tests/inertial_gravity_wave/analysis.py

```diff
@@ -0,0 +1,91 @@
+import datetime
+import warnings
+
+import cmocean  # noqa: F401
+import matplotlib.pyplot as plt
+import numpy as np
+import xarray as xr
+
+from polaris import Step
+from polaris.ocean.tests.inertial_gravity_wave.exact_solution import (
+    ExactSolution,
+)
+
+
+class Analysis(Step):
+    """
+    A step for analysing the output from the inertial gravity wave
+    test case
+
+    Attributes
+    ----------
+    resolutions : list of int
+        The resolutions of the meshes that have been run
+    """
+    def __init__(self, test_case, resolutions):
+        """
+        Create the step
+
+        Parameters
+        ----------
+        test_case : polaris.ocean.tests.inertial_gravity_wave.convergence.Convergence # noqa: E501
```

Comment:
```suggestion
        test_case : polaris.TestCase
```
I think this needs to be more general because technically any case could use this step